README.md

```diff
@@ -319,38 +315,19 @@ All send functions support `SendOptions` as an optional parameter. The `contentT
 
 To learn more about content types, see [Content types with XMTP](https://xmtp.org/docs/concepts/content-types).
 
-Support for other types of content can be added by registering additional `ContentCodecs` with the `Client`. Every codec is associated with a content type identifier, `ContentTypeId`, which is used to signal to the client which codec should be used to process the content that is being sent or received.
-
-<!--not sure about the Codecs link - WDYT?-->
-
-For example, see the [Codecs](https://github.com/xmtp/xmtp-react-native/blob/main/src/lib/CodecRegistry.ts) available in `xmtp-react-native`.
+For example, see the [Codecs](https://github.com/xmtp/xmtp-react-native/blob/main/src/XMTP.types.ts) available in `xmtp-react-native`.
```

Comment:
Link correct?

Reply:
Sort of it's the content types that we automatically support in react native. React Native is the only SDK that default registers all the codecs. You can see which ones are supported here https://github.com/xmtp/xmtp-react-native/blob/main/android/src/main/java/expo/modules/xmtpreactnativesdk/wrappers/ContentJson.kt#L43-L53 and here https://github.com/xmtp/xmtp-react-native/blob/main/ios/Wrappers/DecodedMessageWrapper.swift#L35-L48 where we pre register the codecs. It might be worth linking out to the android and ios codecs inside of the other repos as that is actually where the codec code lives so like here: https://github.com/xmtp/xmtp-android/tree/main/library/src/main/java/org/xmtp/android/library/codecs